src/backend/database_migrations/versions/20221201_174505_add_last_updated_lineages.py

```diff
@@ -22,10 +22,16 @@ def upgrade():
     )
 
     conn = op.get_bind()
+
+    # this is the same as jess' query from backfill sample lineages,
+    # only tacking on the last_updated portion to the query.
+    # this is still valid at this time because we are still
+    # populating the pathogen_genome fields from the pangolin save script.
+
     insert_accessions_sql = sa.sql.text(
         """
-        INSERT INTO aspen.sample_lineages (last_updated)
-        SELECT pg.pangolin_last_updated
+        INSERT INTO aspen.sample_lineages (sample_id, lineage_type, lineage, lineage_probability, lineage_software_version, raw_lineage_output, last_updated)
+        SELECT s.id, 'PANGOLIN', pg.pangolin_lineage, pg.pangolin_probability, pg.pangolin_version, pg.pangolin_output, pg.pangolin_last_updated
```

Comment:
@jgadling is it ok that i'm modifying the previous migration that failed or should i create a new migration file? Also if you know a better way to do this let me know

Reply:
I think you need to update the existing sample lineages rows to include the new field?

Reply:
yes i updated the query

Reply:
fyi @vincent-czi this is the update query